src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -1033,6 +1033,11 @@ public sealed partial class SqlConnectionStringBuilder : System.Data.Common.DbCo
         [System.ComponentModel.DisplayNameAttribute("Host Name In Certificate")]
         [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
         public string HostNameInCertificate { get { throw null; } set { } }
+        /// <include file='../../../../doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml' path='docs/members[@name="SqlConnectionStringBuilder"]/ServerCertificate/*'/>
```

Comment:
Please include changes in netfx > ref project as well. Also make sure you do add this property description in `doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml`. 

I think it also got missed for HostNameInCertificate, and shows blank here: https://learn.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnectionstringbuilder?view=sqlclient-dotnet-standard-5.0
Please fix that too.